scipy/spatial/transform/rotation.py

```diff
@@ -1,12 +1,55 @@
 from __future__ import division, print_function, absolute_import
 
+__all__ = ['Rotation']
```

Comment:
It seem like this `__all__` typically is not necessary. I suggest not to include lines without strong reasons.